### DISCUSSION TITLE
Failed to call service vacuum/send_command. No response from the device

### DISCUSSION THREAD

**Discussion by amjidkhan1** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#730] [Q&A] [answered]
Sorry if this isn't right place for this but I'm not sure what to ask.  

I have Dream D10 Plus, is this supported with this card?

The card is amazing and the Zoom feature works, however if i select a room as per below 

I get 'Failed to call service vacuum/send_command. No response from the device.

I'm not an expert here so not sure if im doing something wrong, the card is amazing and generally works only issue is the room function doesn't work and since my wife has a home salon i want her to quickly use this selection after every client or even automate it. So right now the HA Integration isn't working :( 

Would appreciate any help.  Thank you

      - name: Rooms
        icon: mdi:floor-plan
        run_immediately: false
        coordinates_rounding: true
        coordinates_to_meters_divider: 1000
        selection_type: ROOM
        id_type: number
        max_selections: 999
        repeats_type: EXTERNAL
        max_repeats: 3
        service_call_schema:
          service: vacuum.send_command
          service_data:
            command: app_segment_clean
            entity_id: '[[entity_id]]'
            params:
              - segments: '[[selection]]'
                repeat: '[[repeats]]'
        predefined_selections:
          - id: '1'
            icon:
              name: mdi:toilet
              x: 2400
              'y': 2150

**Comment by PiotrMachowski** [comment]
Which vacuum integration do you use?

**Reply by amjidkhan1** [reply]
Hi, sorry I missed this.  I am using custom:xiaomi-vacuum-map-card
vacuum_platform: Tasshack/dreame-vacuum

**Reply by PiotrMachowski** [reply, marked as answer]
So you have clicked "Generate static config" and only then changed the vacuum platform. The easiest fix is to configure the card again, but without clicking "Generate static config" button.

**Reply by amjidkhan1** [reply]
Oh! Yes I changed the platform after as I missed it first time around. ok ill configure this again tomorrow and hope that resolves the issue. Thank you so much.

**Reply by amjidkhan1** [reply]
thank you it works beautifully!! and looks awesome

**Reply by PiotrMachowski** [reply]
You're welcome ;)
